cohorts/io/gcloud_storage.py

```diff
@@ -64,10 +64,10 @@ def client(self):
         return self._client
 
 
-    def list_files(self, gsuri):
+    def list_files(self, gsuri, **kwargs):
         bucket_name, gs_rel_path = self.parse_uri(gsuri)
         bucket = self._client.get_bucket(bucket_name)
-        return bucket.list_blobs()
+        return bucket.list_blobs(**kwargs)
```

Comment:
Just curious: when/for what you need to pass additional arguments to the `list_blobs`?  (Never mind - just saw your comment in the other tab). Nice addition here, thanks Jacki!